core/templates/components/entity-creation-services/exploration-creation.service.ts

```diff
@@ -116,6 +116,11 @@ angular.module('oppia').factory('ExplorationCreationService', [
           // This callback is triggered when the Cancel button is
           // clicked. No further action is needed.
         });
+      },
+      createNewExplorationNode: function(title: string) {
```

Comment:
If you are going to introduce this new method then please add this in a  *backend-api.service ([optional] use the new backend-api.service on line 51 of this file.) 

Reason: We expect this PR to move HTTP requests to *backend-api.service